Update manifests: 86Box.86BoxManager version 1.7.4

Dragon1573 opened the pull request:
## Description

- Change `PackageName` of the package.

## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- Resolve #162018

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162058)

Dragon1573 committed:
Update manifests: 86Box.86BoxManager version 1.7.4

- Change `PackageName` of the package.

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Trenly commented on manifests/8/86Box/86BoxManager/1.7.4/86Box.86BoxManager.locale.en-US.yaml:
```diff
@@ -5,7 +5,7 @@ Publisher: 86Box Project
 PublisherUrl: https://86box.net
 PublisherSupportUrl: https://paypal.me/86Box
```
I know it wasn't part of the initial change, but this doesn't look like a valid support URL

stephengillie replied:
I've added `paypal` to my "word filter list", to help mitigate this in the future.

stephengillie replied:
Hi @Dragon1573,

Just commenting out the `PublisherSupportUrl` would probably be enough here.

Dragon1573 replied:
```suggestion
# PublisherSupportUrl:
```

Trenly commented:
[Policy] Needs author Feedback

Dragon1573 committed:
Apply suggestions from code review

- Publisher support URL is invalid for getting support.
- It seems to be a "sponsor URL".
- Just comment it out as instructed by @stephengillie

stephengillie commented:
- https://github.com/microsoft/winget-pkgs/issues/142854
